staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go

```diff
@@ -209,7 +209,7 @@ func (o *ScaleOptions) RunScale() error {
 	}
 
 	infos := []*resource.Info{}
-	err = r.Visit(func(info *resource.Info, err error) error {
+	_ = r.Visit(func(info *resource.Info, err error) error {
```

Comment:
copying comment from earlier PR...

ignoring the error here seems like a mistake. was that intentional or should we return early if one of the objects could not be resolved?